common/elasticsearch/elasticsearch.go

```diff
@@ -115,14 +120,6 @@ func CreateElasticSearchConfig(uri *url.URL) (*ElasticSearchConfig, error) {
 		startupFns = append(startupFns, elastic.SetHealthcheck(healthCheck))
 	}
 
-	if len(opts["sniff"]) > 0 {
```

Comment:
Sniff is enabled by default, However it doesn't supported by AWS ES Services (see https://github.com/kubernetes/heapster/issues/1242). Therefore, in case of usage of AWS I automatically disabled this feature.